lib/layout/adaptive.dart

```diff
@@ -5,14 +5,14 @@
 import 'package:flutter/material.dart';
 import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
 
-/// Returns a boolean by screen size.
+/// Returns a boolean value whether the window is considered medium or large size.
 ///
 /// Returns a boolean if we are on a medium or larger screen. Used to
```

Comment:
"The returns a boolean if we are..." comments are now redundant so they can be removed

Reply:
Yes, you are right. I resolved it